Command/ClearInvalidRefreshTokensCommand.php

```diff
@@ -13,14 +13,19 @@
 
 use Gesdinet\JWTRefreshTokenBundle\Model\RefreshTokenInterface;
 use Gesdinet\JWTRefreshTokenBundle\Model\RefreshTokenManagerInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
 
+#[AsCommand(name: 'gesdinet:jwt:clear', description: 'Clear invalid refresh tokens.')]
 class ClearInvalidRefreshTokensCommand extends Command
 {
+    /**
+     * @deprecated
```

Comment:
Attributes are PHP 8+. Does this mean the package will drop support for PHP 7.4?

Reply:
No need, attributes do not break on older versions, they are not taken into account

Reply:
that'll bite you on PHP 7.4 since it can't parse the use Symfony\Component\Console\Attribute\AsCommand due to syntax

Reply:
The problem is that 5.4 does have the Attribute:

https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Console/Attribute/AsCommand.php

And that does support 7.4 :)

Reply:
The attribute syntax was explicitly chosen because it is parsed as a comment on PHP 7 and earlier.  Adding attributes to a class will not break PHP 7 compatibility.  The only thing this approach does is cause duplication of information in order to keep the command class lazy-loaded.

Reply:
Yeah, I just had a weird thing happening where I tried to import `AsCommand` as well on Api-Platform to fix the deprecation there and it basically blew up in my face complaining about the file (assuming cause of the constructor syntax) so I'm just a bit surprised it doesn't seem to do it here/ on the Lexik one.
Maybe I'll try to dig into that some more, just strikes me as odd.